sql/moz-fx-data-shared-prod/fenix_derived/meta_attribution_country_counts_v1/query.sql

```diff
@@ -1,25 +1,54 @@
+{% if is_init() %}
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    CASE
+      WHEN (metrics.string.meta_attribution_app = '382348575493443')
+        THEN 'Firefox Focus for Android'
+      WHEN (metrics.string.meta_attribution_app = '979253712091854')
+        THEN 'Firefox Browser for Android'
+      WHEN (metrics.string.meta_attribution_app = '697946762208244')
+        THEN 'Firefox Nightly for Android'
+      ELSE NULL
+    END AS meta_attribution_app,
+    normalized_channel,
+    metadata.geo.country AS country,
+    COUNT(*) AS ping_count
+  FROM
+    fenix.first_session
+  WHERE
+    DATE(submission_timestamp) >= '2023-11-01'
```

Comment:
Yes, that's valid.
I figured it might be easiest just copying the init.sql logic over, and skip optimizing it as part of this migration effort.

Reply:
Ok that's fine.  I can understand just committing the automated migrated queries since you need to go through every init file.  I would prefer cleaning up the queries now since it will make editing the queries in the future easier and also make the diff for these changes easier to read but that might take too long to do.

Reply:
Alright, I updated some queries to reuse shared logic were it was obvious